IBandPlayerDemo/Assets.xcassets/iconHDOn.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "iconHDOn.png",
```

Comment:
Do not support plain videos on the demo, remove all HD and 360 icons

Reply:
done